dev/io.openliberty.microprofile.lra.1.0.internal/resources/io/openliberty/microprofile/lra/internal/resources/Lra.nlsprops

```diff
@@ -20,3 +20,8 @@
 LRA_INVALID_COORDINATOR_URI=CWMRX5000E: The LRA coordinator URI is invalid
 LRA_INVALID_COORDINATOR_URI.explanation=LRA participants must be able to contact the coordinator
 LRA_INVALID_COORDINATOR_URI.useraction=Correct the LRA coordinator configuration in server.xml
+
+
+LRA_CANT_REGISTER_FILTERS=CWMRX5001E: An error occurred during initialization of the mpLRA-1.0 feature. The error is {0}
```

Comment:
It's not required but often helpful to include the message code (`CWMRX5001E`) in the message key (currently `LRA_CANT_REGISTER_FILTERS`), so that you can search the workspace for the message code and find out where it's produced.